modules/sdk-coin-ada/package.json

```diff
@@ -43,7 +43,11 @@
   "dependencies": {
     "@bitgo/sdk-core": "^1.0.1",
     "@bitgo/statics": "^6.17.0",
-    "bignumber.js": "^9.0.2"
+    "@emurgo/cardano-serialization-lib-asmjs": "^10.2.0",
+    "@emurgo/cardano-serialization-lib-browser": "^10.2.0",
+    "@emurgo/cardano-serialization-lib-nodejs": "^10.2.0",
+    "bignumber.js": "^9.0.2",
```

Comment:
do we need bignumber and base58 module?

Reply:
BigNumber is part of the default skeleton for TransactionBuilder, but we can remove base58